srml/staking/src/phragmen.rs

```diff
@@ -24,6 +24,7 @@ use crate::{Exposure, BalanceOf, Trait, ValidatorPrefs, IndividualExposure};
 
 type Fraction = PerU128;
 type ExtendedBalance = u128;
+const SCALE_FACTOR_64: u128 = u64::max_value() as u128;
```

Comment:
if you make this `u64::max_value() as u128 + 1`, then it'll become a simple shift left and shift right in the code which should be a lot faster in wasm.

Reply:
```suggestion
const SCALE_FACTOR_64: u128 = (u64::max_value() as u128) + 1;
```